src/middleware/webAuth.ts

```diff
@@ -56,13 +55,10 @@ async function webAuth(request, response, next) {
 	return next();
 }
 
-function isStartOfPath(path, value) {
+function isStartOfPath(path: string, value: string): boolean {
 	return path.indexOf(value) === 0;
 }
 
-BigInt.prototype['toJSON'] = function () {
-	return this.toString();
```

Comment:
I removed it as I couldn't see any occurrences of `toJSON` used on a `BigInt` but I can put it back in as I imagine it's used under the hood when we're serialising some other object (Not sure which though? AFAIK we don't specify `BigInt` anywhere and the only cases of `toJSON` I remember seeing were in serialising models to put into redis)

Reply:
I'm trying to work my way through everything here, but I just moved this week so I apologize for the delay in my review. The account info that we receive via gRPC returns a BigInt for one of the fields. Since we pass this onto the profile page, it would cause a crash if this polyfill wasn't present.

Reply:
Is `webAuth` the right place for this then? Will stick it back in in the meantime

Reply:
Also, if it's about retrieving a `BigInt` from gRPC, wouldn't it be about parsing it _from_ JSON instead? Or am I misinterpreting?